File: Python/Problem_7.py
def isprime(n):
	for x in range(2, int(n**0.5)+1):
		if n%x==0: return False
	return True
	
def findprime(n):
	prime = 0
	count = 0
	x = 3 #so we can increment by 2 to make the loop run faster
	
	while count < n-1: #n-1 because we start at 3
			if(isprime(x)):
				solution = x
				count+=1
			x+=2 
			
	return prime
	
print findprime(10001)

File: C++/Problem 7.cpp
#include <iostream>
#include <vector>
#include <math.h>


bool isPrime(int i) {

	if(i == 0 || i == 1) return 0;

	
	for(int divisor=3; divisor<i-1; divisor++) {

		if(i % divisor == 0) return false;
	}

	return true;
}
	

int main() {

	int highestprime = 0;

	int i = 0;

	for(int x=0; i<10001; x++) {
		if(isPrime(x)) {
			std::cout << "prime: " << x << "   count: " << i << " \n";
			highestprime = x;
			i++;
		}
	}


	printf("%i\n", highestprime);

	std::cin.get();
		
	return 0;
}
